deploy/kubernetes/controller.yaml

```diff
@@ -0,0 +1,85 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-gce-pd
+  labels:
+    app: csi-gce-pd
+spec:
+  selector:
+    app: csi-gce-pd
+  ports:
+    - name: dummy
+      port: 12345
```

Comment:
nit: add a comments explaining the purpose at the top of the file, e.g.:

```YAML
# The dummy service is required for creating a StatefulSet
```

Reply:
done

Reply:
I'm not sure you actually need a service object. I launched the plugin without the service object and it was fine

Reply:
weird, we've had this discussion before and I remember trying specifically and having it fail. I will test again